packages/core/phase2/createPncDisposal.ts

```diff
@@ -43,7 +43,7 @@ const createPncDisposal = (
     ),
     text: disposalText,
     type: pncDisposalType
-  } as PncDisposal
```

Comment:
The changes in the `PncQueryResult` schema are related to removing this type casting.
- qtyMonetaryValue is string in the schema, but we were assign a number value here.
- `sentenceDate` and `type` can be undefined in Phase 2 but are required fields in the schema. To bypass ts error we were force casting to `PncDisposal` type to bypass the ts error.